modules/ROOT/pages/6.6.1-release-notes.adoc

```diff
@@ -66,7 +66,15 @@ The {productname} 6.6.1 release includes an accompanying release of the **Enhanc
 ==== Context menu and context toolbar no longer show when the selected image is in a non-editable context
 //#TINY-10016
 
-// CCFR here.
+When a {productname} instance is setup with `editable_root: false` set, the entire instance is, by default, a read-only instance. To allow editing of any portion, that portion must have a `contenteditable="true"` attribute set.
+
+Previously, however, if an image was in a {productname} instance with an `editable_root: false` set, bringing up the {productname} context menu (eg by right-clicking on the image in a desktop setting or by long-pressing on the image in a mobile setting) allowed the {productname} context toolbar to be further presented, which allowed for image manipulation.
+
+**Enhanced Image Editing** 1.1.1 addresses this. With this update, right-clicking on an image, or long-pressing on an image in a mobile setting, no longer brings up the {productname} context menu, obviating the path to, further, bringing up the {productname} toolbar.
```

Comment:
This is not completely accurate. The enhanced image editing plugin fixes the context toolbar showing up. And the normal image plugin fixes the context menu showing up.

Reply:
to quote my commit message:

> Corrections and re-write consequent to developer notes.